.env.development

```diff
@@ -8,4 +8,4 @@ NEXT_PUBLIC_API_URL=http://localhost:8001/api/v1
 NEXT_PUBLIC_PI_SDK_URL=https://sdk.minepi.com/pi-sdk.js
 
 NEXT_PUBLIC_SENTRY_DSN="ADD YOUR SENTRY DSN"
-NEXT_PUBLIC_IMAGE_PLACEHOLDER_URL="ADD YOUR IMAGE PLACEHOLDER URL"
+NEXT_PUBLIC_IMAGE_PLACEHOLDER_URL='/path/to/default/image.png'
```

Comment:
Can you please stop touching this @adisa39..? 😅

Reply:
Noted